java-2d-array

```diff
@@ -0,0 +1,44 @@
+import java.io.*;
+import java.math.*;
+import java.security.*;
+import java.text.*;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.regex.*;
+
+public class Solution {
+
+
+
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
also noting that the coding conventions in java declares that the static final variables should be written in snake case with only upper-case characters :)